### PR TITLE
Change help menu to be the same style as the user menu

### PR DESCRIPTION
**Pull request by bethanyaconnor**
[Jira](https://codedotorg.atlassian.net/browse/FND-447)

Screenshot 2019-06-17 at 3 57 32 PM

### CODE REVIEW

File: shared/css/user-menu.scss [hunk 1]
```diff
@@ -72,6 +72,61 @@
...
+  &.show-always {
+    display: block;
+  }
+  .show-mobile {
+    display: none;
+  }
+  display: inline-block;
+  float: left;
+  z-index: 99;
+  position: relative;
```
Comment (Erin007) [inline shared/css/user-menu.scss:96]: tiny nit: should these lines be indented?

**Review by Erin007 — APPROVED** [review]
code changes look good. Just double check that the styling looks exactly as you expect on both dashboard and pegasus, mobile and non-mobile, please, since the base styles can frustratingly differ slightly.

File: shared/css/user-menu.scss (hunk 1 above)
Comment (bethanyaconnor) [reply shared/css/user-menu.scss:96]: I don't think so but I'll add an extra new line to separate them from all the unrelated stuff right above.

**Comment by bethanyaconnor** [comment]
Thanks for saying that! I did a last check and it turns out I broke hiding the help button on mobile 🙈

**Review by Erin007 — APPROVED** [review]

**Pull request merged**